grr/server/grr_response_server/gui/ui/components/timestamp/relative_timestamp_pipe.ts

```diff
@@ -0,0 +1,54 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {DateTime} from 'luxon';
+
+@Pipe({
+  name: 'relativeTimestamp'
+})
+export class RelativeTimestampPipe implements PipeTransform {
+  readonly oneDay = 86400000; // One day in millis
```

Comment:
These are de-facto constants. Please us `static` for constants - they are not changing per instance of RelativeTimestampPipe. 

Additionally, if class members (variables or methods) are not used outside of the class, prefer to make them private to prevent outside classes using/depending on them. 

Finally, you could move the information that the comment conveys ("in millis") into the variable name. This removes the need for the reader to read the comment, because the variable conveys all information:

```
  private static readonly ONE_DAY_IN_MILLIS = 86400000;
```

Reply:
Good idea!